Python/hamt.c

```diff
@@ -1194,7 +1194,7 @@ hamt_node_bitmap_dump(PyHamtNode_Bitmap *node,
             }
         }
 
-        if (PyUnicodeWriter_WriteUTF8(writer, "\n", 1) < 0) {
+        if (PyUnicodeWriter_WriteASCII(writer, "\n", 1) < 0) {
```

Comment:
Is it more efficient to use WriteASCII or WriteChar here?

Reply:
This code is only built in debug mode. I prefer to keep WriteASCII() to make the code more consistent. It's not performance sensitive, it's only used for debugging purpose.